src/store/modules/recipeStore.js

```diff
@@ -68,6 +68,20 @@ export const recipeStore = {
           context.commit("SET_LOADING_STATE", false);
         });
     },
+
+    async fetch_recipe_raw(context, recipeId) {
+      const uri = this.state.named_urls.recipe_resources + "/" + recipeId;
+      try {
+        const response = await this.state.api.client.get(
+          uri,
+          this.state.api.options
```

Comment:
This api.options seem to be flakky sometimes. we'd look into it later in the mean time, se the headers manually in the request like you've done in other places before.

```
{
    headers: { Authorization: `Bearer ${process.env.DEV_TOKEN}` },
}
```

Reply:
Done